backend/routes/brain_routes.py

```diff
@@ -109,29 +109,6 @@ async def get_brain_endpoint(
         return {"error": f"No brain found with brain_id {brain_id}"}
 
 
-# delete one brain
-@brain_router.delete(
```

Comment:
Where is the /brains/{brain_id}/subscription endpoint in the routes folder ?

Reply:
This PR's base is `shareable-brain-7.1`, which introduces the delete endpoint

Plus, you've migrated us to `/brain/{brain_id}/subscription`... 🙈 (without the 's')

Reply:
Yes, but this is going to break between 7.1 and 7.2 :/

Reply:
My PR is changing everything back to /brains no worries, you can use it with or without the s